README.md

```diff
@@ -1,32 +1,21 @@
-# Kotlin cross-platform / multi-format reflectionless serialization
+# Kotlin multiplatform / multi-format serialization reflectionless serialization
 
 [![JetBrains incubator project](https://jb.gg/badges/incubator.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](http://www.apache.org/licenses/LICENSE-2.0)
 [![TeamCity build](https://img.shields.io/teamcity/http/teamcity.jetbrains.com/s/KotlinTools_KotlinxSerialization_Ko.svg)](https://teamcity.jetbrains.com/viewType.html?buildTypeId=KotlinTools_KotlinxSerialization_Ko&guest=1)
 [![Download](https://api.bintray.com/packages/kotlin/kotlinx/kotlinx.serialization.runtime/images/download.svg) ](https://bintray.com/kotlin/kotlinx/kotlinx.serialization.runtime/_latestVersion)
 
-Kotlin serialization consists of a compiler plugin, which automatically produces visitor code for classes, and runtime library, which uses generated code to serialize objects without reflection.
+Kotlin serialization consists of a compiler plugin, that generates visitor code for serializable classes,
+ runtime library with core serialization API and JSON format, and support libraries with ProtoBuf, CBOR and properties formats.
```

Comment:
WDYT of adding the corresponding links to `formats/README.md#protobuf` etc?

Reply:
Not now, because it's not really that useful

Reply:
Why not useful?

Reply:
Because it contains no really useful information.

Reply:
It contains artifact-id for protobuf and cbor modules (and basically, says that they **are** separate artifacts, instead of vague 'support libraries')